stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -199,15 +199,31 @@ convert(::Type{Adjoint{T,S}}, A::Adjoint) where {T,S} = Adjoint{T,S}(convert(S,
 convert(::Type{Transpose{T,S}}, A::Transpose) where {T,S} = Transpose{T,S}(convert(S, A.parent))
 
 # Strides and pointer for transposed strided arrays — but only if the elements are actually stored in memory
-Base.strides(A::Adjoint{<:Real, <:AbstractVector}) = (1, stride(A.parent, 1))
-Base.strides(A::Transpose{<:Any, <:AbstractVector}) = (1, stride(A.parent, 1))
+Base.strides(A::Adjoint{<:Real, <:AbstractVector}) = (stride(A.parent, 2), stride(A.parent, 1))
+Base.strides(A::Transpose{<:Any, <:AbstractVector}) = (stride(A.parent, 2), stride(A.parent, 1))
 # For matrices it's slightly faster to use reverse and avoid calling stride twice
 Base.strides(A::Adjoint{<:Real, <:AbstractMatrix}) = reverse(strides(A.parent))
 Base.strides(A::Transpose{<:Any, <:AbstractMatrix}) = reverse(strides(A.parent))
 
 Base.unsafe_convert(::Type{Ptr{T}}, A::Adjoint{<:Real, <:AbstractVecOrMat}) where {T} = Base.unsafe_convert(Ptr{T}, A.parent)
 Base.unsafe_convert(::Type{Ptr{T}}, A::Transpose{<:Any, <:AbstractVecOrMat}) where {T} = Base.unsafe_convert(Ptr{T}, A.parent)
 
+"""
+    ConjPtr{T}
+
+A memory address referring to the complex conjugate of data of type T. However, there is no guarantee that the memory is actually valid, or that it actually represents data of the specified type.
+"""
+struct ConjPtr{T} 
```

Comment:
What issue does this fix? Since it has a docstring it looks like a new feature which we shouldn't put into 1.5.

Reply:
The issue is in the tests:
```julia
@test pointer(view(B',:,:)') == pointer(B)
```
I'll remove it though since it sounds like it will hold things up.